languages/common-lisp/reference/README.md

```diff
@@ -111,10 +114,10 @@ as template.
   - [Structures][struct]
   - Symbols
 
-## Exercise Concepts
-| Concept | Learning Objectives |
-| ------- | ------------------- |
-| [`sexpr`][sexpr] | Teach the student about S-expressions, the base unit of all Lisp code. The student should be able to distinguish an `atom` from a `cons` and know how to access the `car` and `cdr` of a cons pair. |
+## Implemented Concept Exercises
+| Exercise           | Concepts                                           |
+| ------------------ | -------------------------------------------------- |
+| [`basics`][basics] | `comments`, `expressions`, `cons.basic`, `symbols` |
```

Comment:
We'll be updating the guidelines on the concepts a bit. You can anticipate the changes as follows:

```suggestion
| [`basics`][basics] | `basics` |
```

Reply:
Hmm, okay. Will all of the exercises just have their own name as a concept then? And no longer multiple concepts to an exercise?